src/Sarif/RuntimeConditions.cs

```diff
@@ -26,10 +26,15 @@ public enum RuntimeConditions : uint
         //LoggerRaisedExceptionOnClose,
 
         // Fatal conditions
-        ExceptionInstantiatingSkimmers = 0x01,
-        ExceptionInSkimmerInitialize = 0x02,
-        ExceptionRaisedInSkimmerCanAnalyze = 0x04,
-        ExceptionInSkimmerAnalyze = 0x08,
+        // InvalidCommandLineOption is a useful bit to 
+        // use as value 0x1. This allows for a consistent 
+        // return value in cases where a command-line argument
+        // for --rich-return-code can't be detected because it 
+        // fails to parse. 
+        InvalidCommandLineOption = 0x1,
```

Comment:
>InvalidCommandLineOption [](start = 8, length = 24)

The argument makes sense. Are you comfortable you're not breaking anyone who might be relying on a rich return code where 0x01 meant "ExceptionInstantiatingSkimmers"?

Wildly unlikely, and I'll approve -- just wanted to make sure you considered it.